website/docs/cli/import/index.mdx

```diff
@@ -9,72 +9,16 @@ description: >-
-=======
-Terraform can import existing infrastructure resources. This functionality allows you take
-resources you created by some other means and bring them under Terraform
-management.
-=======
-Terraform can import existing infrastructure resources. This functionality allows you take resources you created by some other means and bring them under Terraform
-management.
+~> **Warning:** Terraform expects that each remote object is bound to only one resource address. You should import each remote object to only one Terraform resource address. If you import the same object multiple times, Terraform may exhibit unwanted behavior. Refer to [State](/language/state) for more details.
 
-This method lets you slowly transition infrastructure to Terraform, or
-to be able to be confident that you can use Terraform in the future if it
-potentially doesn't support every feature you need today.
+## State Only
 
-~> Warning: Terraform expects that each remote object it is managing will be
-bound to only one resource address, which is normally guaranteed by Terraform
-itself having created all objects. If you import existing objects into Terraform, be careful to import each remote object to only one Terraform resource address. If you import the same object multiple times, Terraform may exhibit unwanted behavior. Refer to [State](/language/state) for more details.
->>>>>>> parent of 84edd84471 (more content updates for flow)
-
-This is a great way to slowly transition infrastructure to Terraform, or
-to be able to be confident that you can use Terraform in the future if it
-potentially doesn't support every feature you need today.
-
-<<<<<<< HEAD
-~> Warning: Terraform expects that each remote object it is managing will be
-bound to only one resource address, which is normally guaranteed by Terraform
-itself having created all objects. If you import existing objects into Terraform,
-be careful to import each remote object to only one Terraform resource address.
-If you import the same object multiple times, Terraform may exhibit unwanted
-behavior. For more information on this assumption, see
-[the State section](/language/state).
->>>>>>> parent of 0a7e221a49 (Remove future-facing statements)
-
-## Currently State Only
-
-<<<<<<< HEAD
 Terraform import can only import resources into the [state](/language/state). Importing does not generate configuration.
-=======
-Terraform import can only import resources into the [state](/language/state). It does not generate configuration.
->>>>>>> parent of 84edd84471 (more content updates for flow)
 
-Because of this, prior to running `terraform import` you must manually write a `resource` configuration block for the resource that describes where Terraform should map the imported object.
+Before you run `terraform import` you must manually write a `resource` configuration block for the resource. The resource block describes where Terraform should map the imported object.
```

Comment:
I'm sorry about this. I completely messed this page up when I accidentlly pushed some commits to main and had to revert them. This is not that different from the original (not as much was removed as the PR is making it look.
<img width="920" alt="Screen Shot 2022-10-24 at 5 35 31 PM" src="https://user-images.githubusercontent.com/83350965/197634208-481ca27b-d1f7-4142-8284-b11b79a5e66a.png">